### DISCUSSION TITLE
Add a way to display beatmap packs separate from each other

### DISCUSSION THREAD

**Discussion by mas-alone** [ppy/osu#13199] [Ideas]
**Describe the new feature:** 
Added a "pack" category to the beatmap, because when players have a lot of beatmaps, "packs" are mixed with regular beatmaps, making it very difficult to find, sometimes if you want to play some "packs" it is hard to find if you don't remember the name. It would be great to add a "pack" to put all automatic categories with packs in their titles.
![123pakc](https://user-images.githubusercontent.com/52590027/118015112-83040380-b386-11eb-8bcb-efad5b271a02.png)

**Comment by peppy** [comment]
I'm not sure if this fits in the location you propose it. We created the playlist system as a way to hopefully surface "packs" in the future. To make something like this work at some select there would need to be tags/metadata provided by the server to actually know the relationship beatmaps have to packs.

I'm not sure enough people use packs to warrant this, but worth keeping this open to see if there's enough interest for a far-future implementation.
